pkg/util/strings/BUILD

```diff
@@ -13,7 +13,7 @@ go_library(
         "line_delimiter.go",
         "strings.go",
     ],
-    importpath = "k8s.io/kubernetes/pkg/util/strings",
+    importpath = "k8s.io/kubernetes/util/strings",
```

Comment:
pretty sure this change is wrong.

Reply:
we should be removing this package instead.

Reply:
right. this should be `importpath = "k8s.io/util/strings",`

Reply:
Pretty sure package should be `"k8s.io/kubernetes/pkg/util/strings"` like it was, but if we want to switch everything to `k8s.io/utils/strings` we can remove this whole directory and just change the imports everywhere else.

Reply:
yes, that was the intention of the issue. looks like i was not clear enough.

Reply:
Thanks @dims . While I cannot find either the k8s.io/utils/strings package nor the k8s.io/kubernetes/pkg/util/strings. Am I missing anything here?

Reply:
github url : https://github.com/kubernetes/utils/tree/master/strings
package name : k8s.io/utils/strings

Reply:
Thanks. I see. So it's under another repo. And the original package was under this repo's pkg/util/strings, which has already gone, right?

Reply:
@chaopli yes pull from new repo. looks like your CLA is not setup right as well. please look at tips and contact here to resolve it:
https://github.com/kubernetes/community/blob/master/CLA.md#troubleshooting